docs/references/release-notes.md

```diff
@@ -14,12 +189,50 @@
 ### New Contributors
 
 - [@peachkeel](https://github.com/peachkeel) made their first contribution in
-  [#184](https://github.com/Quansight/ragna/pull/244)
+  [#244](https://github.com/Quansight/ragna/pull/244)
 - [@nenb](https://github.com/nenb) made their first contribution in
   [#252](https://github.com/Quansight/ragna/pull/252)
 
 ## Version 0.1.2
 
+### Breaking changes
```

Comment:
While writing the breaking changes for `0.2.0`, I realized this change was never documented. I had it written up anyway, so I put it here. I would also be fine to remove this again to avoid "changing history".